### PR TITLE
[ADD] module to set version on partner from stock picking validation

### PR DESCRIPTION
**Pull request by bguillot**
Hi

This module depends on the module of this PR : OCA/partner-contact#669

It sets a version on addresses of the stock picking when the stock picking is validated.

### CODE REVIEW

File: stock_partner_version/README.rst
Comment (rousseldenis) [inline stock_partner_version/README.rst:1]: It is recommended to use https://github.com/OCA/maintainer-tools#readme-generator to generate README from fragments

File: stock_partner_version/__manifest__.py
Comment (rousseldenis) [inline stock_partner_version/__manifest__.py]: Please change at least this using https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

File: stock_partner_version/__manifest__.py
Comment (rousseldenis) [inline stock_partner_version/__manifest__.py]: Please remove void entries

File: stock_partner_version/models/stock.py
Comment (rousseldenis) [inline stock_partner_version/models/stock.py:14]: This causes errors in other tests as partner_id is not mandatory.

```suggestion
           for picking in self.filtered('partner_id'):
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Some comments

**Comment by rousseldenis** [comment]
@bguillot Do you plan to finish this ?

**Comment by bguillot** [comment]
@rousseldenis Sorry for the late answer, I made the changes.

And thank you for the review !

**Review by rousseldenis — APPROVED** [review]
Code review.

Remove change in oca_dependencies before merge

**Comment by rousseldenis** [comment]
@bguillot Do you plan to squash before merging?

**Pull request closed without merging**
